OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -73,13 +73,11 @@ public class AircraftInfo : PausableConditionalTraitInfo, IPositionableInfo, IFa
 		[VoiceReference]
 		public readonly string Voice = "Action";
 
-		[GrantedConditionReference]
 		[Desc("The condition to grant to self while airborne.")]
-		public readonly string AirborneCondition = null;
+		public readonly GrantedConditionReference AirborneCondition;
```

Comment:
You seem to default to `new GrantedConditionReference()` to replace `null` in some cases but not in all. Can you explain why? (Or if this is an oversight, can we be consistent?)